tests/bsim/bluetooth/mesh/src/test_brg.c

```diff
@@ -512,12 +551,114 @@ static void test_tester_simple(void)
 
 	LOG_INF("Checking data...");
 	for (int i = 0; i < REMOTE_NODES; i++) {
-		ASSERT_OK(send_get(DEVICE_ADDR_START + i));
+		ASSERT_OK(send_get(DEVICE_ADDR_START + i, NULL));
+		ASSERT_OK(k_sem_take(&status_msg_recvd_sem, K_SECONDS(5)));
+
+		ASSERT_EQUAL(recvd_msgs_cnt, 0);
+	}
+
+	PASS();
+}
+
+static void tester_simple_multicast(int test_type)
+{
+	uint8_t status;
+	int err;
+	const int msgs_cnt = 3;
+	uint16_t addr = (test_type == TEST_TYPE_GROUP) ? GROUP_ADDR : va_entry->addr;
+	const uint8_t *uuid = (test_type == TEST_TYPE_VA) ? va_entry->uuid : NULL;
+
+	bt_mesh_test_cfg_set(NULL, WAIT_TIME);
+	bt_mesh_device_setup(&tester_prov, &comp);
+
+	tester_setup(test_type);
+
+	for (int i = 0; i < 1 /* bridge */ + REMOTE_NODES_MULTICAST; i++) {
+		LOG_INF("Waiting for a device to provision...");
+		ASSERT_OK(k_sem_take(&prov_sem, K_SECONDS(40)));
+	}
+
+	tester_bridge_configure(REMOTE_NODES_MULTICAST);
+
+	for (int i = 0; i < REMOTE_NODES_MULTICAST; i++) {
+		tester_device_configure(i + 1, DEVICE_ADDR_START + i, test_type);
+	}
+
+	/* Adding devices to bridge table */
+	for (int i = 0; i < REMOTE_NODES_MULTICAST; i++) {
+		/* Bridge messages from tester to multicast addr, for each subnet expect the last */
+		if (i != REMOTE_NODES_MULTICAST - 1) {
+			bridge_entry_add(PROV_ADDR, addr, 0, i + 1, BT_MESH_BRG_CFG_DIR_ONEWAY);
+		}
+
+		/* Bridge messages from remote nodes to tester */
+		bridge_entry_add(DEVICE_ADDR_START + i, PROV_ADDR, i + 1, 0,
+				 BT_MESH_BRG_CFG_DIR_ONEWAY);
```

Comment:
If you are not sending a unicast "GET" message to the last device according to lines 613-621, then no reason in the reverse entry for the last device. Then we could reuse some code from `test_tester_simple`.

Reply:
Sorry, I'm wrong. I think you need to call `k_sem_take(&status_msg_recvd_sem, K_SECONDS(5)))` after line 621 and expect it to timeout. Then it will guarantee, that the 3rd device didn't receive "GET" message and send the responce back.

Reply:
Isn't the changes on 405 sufficient for making sure of this?

Reply:
Maybe not, otherwise why we need the reverse entry in the bridge table?

Reply:
The reverse entry is there so the nodes are able to send messages back to the tester (va/group addressing requires unidirectional table entries). We could of course configure it so the nodes send messages back by some other means... But I think this is sufficient.

Reply:
Yes, but nothing is sent to that 3rd node that would trigger a response from that node. Either we need to check that nothing comes back from that node, or don't add reverse entry.

Reply:
Oh, sorry, I was confused by the `device_data_cb` name, I though it was callback on the dut side. Then it looks good.